src/main/java/beam/router/r5/R5MnetBuilder.java

```diff
@@ -121,10 +121,6 @@ public void buildMNet() {
             log.warn("Fixed {} links which were having the same `fromNode` and `toNode`", numberOfFixes);
         }
 
-
```

Comment:
I'll defer if anybody else can review this removal?

Reply:
i guess it is not correct, i will bring it back

Reply:
fixed